api/views.py

```diff
@@ -65,8 +65,8 @@ def contact():
 # TODO: Find a better way to seed default data into database.
 @views.route('/temp', methods=['GET'])
 def temp():
-    from api import db, UserModel
-    me = UserModel(username='jventura3', 
+    from api import db, UsersModel
+    me = UsersModel(username='jventura3', 
```

Comment:
See comment on #20 regarding the appropriateness of this route.

Reply:
> See comment on #20 regarding the appropriateness of this route.

@jtanderson Resolved.